### PR TITLE
iOS13 Fix and new plugin project style

### PR DESCRIPTION
**Pull request by Px7-941**
Converted Xamarin Libraries to SDK Style & Multi-Targeted Projects -> [https://montemagno.com](https://montemagno.com/converting-xamarin-libraries-to-sdk-style-multi-targeted-projects/)

The redesign of the simple app is not yet finished and comes in another PR.

Fix for #115

### CODE REVIEW

**Comment by Px7-941** [comment]
iOS 13 fix this backwards compatible.
Was rewritten from Objective-C to C#

The other adjustments to the plugin are not logic changes.
All signatures remain the same.

The nuget now consists of fewer DLLs which increases storage and loading speed.

The solution is much tidier and easier to understand.

Works with Visual Studio Windows and Visual Studio for Mac.

**Comment by rdelrosario** [comment]
Great stuff!! Thanks for contributing

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-## Push Notification Plugin for Xamarin iOS and Android
+## Push Notification Plugin for Xamarin iOS and Android (Updated original package to use secure storage)
 Simple cross platform plugin for handling push notifications. Uses FCM for Android and APS for iOS.
 
 ### Setup
-* Available on NuGet: http://www.nuget.org/packages/Plugin.PushNotification [![NuGet](https://img.shields.io/nuget/v/Plugin.PushNotification.svg?label=NuGet)](https://www.nuget.org/packages/Plugin.PushNotification/)
+* Available on NuGet: https://www.nuget.org/packages/Plugin.PushNotification-updated/1.3.1
```
Comment (rdelrosario) [inline README.md]: Here should be https://www.nuget.org/packages/Plugin.PushNotification

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-## Push Notification Plugin for Xamarin iOS and Android
+## Push Notification Plugin for Xamarin iOS and Android (Updated original package to use secure storage)
```
Comment (rdelrosario) [inline README.md]: We can remove (Updated original package to use secure storage)

File: src/PushNotification/iOS/PushNotificationManager.apple.cs [hunk 3]
```diff
@@ -264,16 +273,20 @@ public void DidReceiveNotificationResponse(UNUserNotificationCenter center, UNNo
...
         {
-            string trimmedDeviceToken = deviceToken.Description;
-            if (!string.IsNullOrWhiteSpace(trimmedDeviceToken))
+            var length = (int)deviceToken.Length;
+            if (length == 0)
+            {
+                return;
+            }
+
+            var hex = new StringBuilder(length * 2);
```
[the remark this answers is not in the document]
Comment (Px7-941) [reply src/PushNotification/iOS/PushNotificationManager.apple.cs]: I just tested it with xCode 10.3, Visual Studo for Mac 8.2.6 and an iPhone 6+ with iOS 12.4.2 and the code works.

This code adjustment is no longer necessary. This code is backwards compatible.

File: nuget/Plugin.nuspec [hunk 4]
```diff
@@ -2,25 +2,28 @@
 <package >
   <metadata minClientVersion="2.8.1">
     <id>Plugin.PushNotification</id>
-    <version>1.3.0</version>
-    <title>Push Notification Plugin for Xamarin</title>
-    <authors>Rendy Del Rosario</authors>
+    <version>1.1.1</version>
+    <title>Push Notification Plugin for Xamarin (Updated)</title>
```
[the remark this answers is not in the document]
Comment (Px7-941) [reply nuget/Plugin.nuspec]: Plugin.nuspec is no longer needed. Can be done in .csproj.

File: README.md (hunk 1 above)
Comment (Px7-941) [reply README.md]: I'll adjust that.

File: README.md (hunk 2 above)
Comment (Px7-941) [reply README.md]: I'll adjust that.

**Pull request merged**